test/_functions.jl

```diff
@@ -257,6 +257,19 @@ end
     @test AxisKeys.hasnames(similar(M, 2,2,2)) == false
 
 end
+@testset "deleteat!" begin
+    kv = KeyedArray([1, 2, 3, 4, 5, 6.0], a=[:a, :b, :c, :d, :e, :f])
```

Comment:
If these construct with wrapdims, then I think they will be tested in both wrapper orders.

Reply:
could you elaborate? not sure I follow

Reply:
The most of the tests are in a loop, which alters the flag `wrapdims` checks to decide whether it makes a NDA(KA(...)) or KA(NDA(...)), to test both ways around.